user/models.py

```diff
@@ -0,0 +1,28 @@
+from django.db import models
+from .validation import validate_account, validate_birthday, validate_email, validate_name, validate_phone
+
+class User(models.Model):
+
+    account                 = models.CharField(max_length = 50, validators = [validate_account], unique = True)
+    password                = models.CharField(max_length = 256)
+    name                    = models.CharField(max_length = 50, validators = [validate_name])
+    phone                   = models.CharField(max_length = 50, validators = [validate_phone])
+    email                   = models.CharField(max_length = 50, validators = [validate_email], unique = True)
+    birthday                = models.CharField(max_length = 50, validators = [validate_birthday])
+    is_sms_marketing_agree  = models.BooleanField(default = False, blank = True)
+    is_email_maketing_agree = models.BooleanField(default = False, blank = True)
+    
+    class Meta:
+        db_table= 'users'
+
+class UserShippingDestination:
```

Comment:
```suggestion
class ShippingDestination:
```
이미 User 앱에 있고, 배송지에 대한 정보에는 유저가 포함되어 있으므로, User라는 단어를 붙이지 않으셔도 좋습니다. 클래스이름이 너무 길어지는 것을 방지하기 위함입니다. 추후 views를 작성할 때 번거롭거나 가독성이 떨어질 수 있습니다 :)